public/index.html

```diff
@@ -1,23 +1,11 @@
 <!DOCTYPE html>
 <html>
-<head>
-    <title>Super Mario</title>
-    <style>
-        body {
-            background: #000;
-            margin: 0;
-        }
-
-        #screen {
-            display: block;
-            image-rendering: pixelated;
-            height: 100vh;
-            margin: auto;
-        }
-    </style>
-    <script type="module" src="/js/main.js"></script>
-</head>
-<body>
-    <canvas id="screen" width="256" height="240"></canvas>
-</body>
+    <head>
+        <title>Super Mario</title>
+        <link rel="stylesheet" href="/css/layout.css">
+        <script type="module" src="/js/main.js"></script>
+    </head>
+    <body>
+        <canvas id="screen" width="256" height="240" data-aspect-ratio="4:3"></canvas>
```

Comment:
I feel like there is a detachment between the data property. It looks like you would be able to write any aspect ratio there.

Reply:
Removed data-attribute, using class now.